src/js/angular/ttyg/directives/chat-panel.directive.js

```diff
@@ -64,10 +64,21 @@ function ChatPanelComponent(toastr, $translate, TTYGContextService, TTYGService)
                 if (!$scope.chatItem.chatId) {
                     createNewChat();
                 } else {
-                    askQuestion();
+                    askQuestion($scope.chatItem);
                 }
             };
 
+            $scope.regenerate = (chatItem) => {
+                const question = getEmptyChatItem();
```

Comment:
This is a ChatItemModel and not a question

Reply:
Renamed.